### ISSUE TITLE
Static Analysis of code

### ISSUE THREAD

**Issue by bagmangood** [appfolio/gemsurance#30]
Gemsurance currently requires both the ruby version and all of the dependencies installed in order to run the check.

Ideally you could run the check with only needing to install `gemsurance`

**Comment by bagmangood** [comment]
yup - that was what I was aiming for. I think it might require a substantial re-write of the internals, since right now gemsurance leverages `bundler`'s code for wrapping all of the calls to rubygems.

**Comment by jonkessler** [comment]
I think that could make sense as an additional use case, but I wouldn’t replace the way it’s currently done.
